integration-tests/actions/vrf/common/actions.go

```diff
@@ -216,3 +220,37 @@ func CreateVRFKeyOnVRFNode(vrfNode *VRFNode, l zerolog.Logger) (*client.VRFKey,
 		Msg("VRF Key created on the Node")
 	return vrfKey, pubKeyCompressed, nil
 }
+
+func FundNodesIfNeeded(ctx context.Context, existingEnvConfig *vrf_common_config.ExistingEnvConfig, client blockchain.EVMClient, l zerolog.Logger) error {
+	if *existingEnvConfig.NodeSendingKeyFundingMin > 0 {
+		for _, sendingKey := range existingEnvConfig.NodeSendingKeys {
+			address := common.HexToAddress(sendingKey)
+			sendingKeyBalance, err := client.BalanceAt(ctx, address)
+			if err != nil {
+				return err
+			}
+			fundingAtLeast := conversions.EtherToWei(big.NewFloat(*existingEnvConfig.NodeSendingKeyFundingMin))
+			fundingToSendWei := new(big.Int).Sub(fundingAtLeast, sendingKeyBalance)
```

Comment:
`sendingKeyBalance` can be larger than `fundingAtLeast`, in that case, we don't need to fund it.

Reply:
but current code does that - it gets the difference between `fundingAtLeast` and `sendingKeyBalance` and if that difference is positive value, than we fund sending keys, if not - we dont fund

Reply:
Okay, just pointing out that if `fundingToSendWei` is a negative value, there is no point in proceeding with any further conversions.